files/system/oem/09_sshd.yaml

```diff
@@ -25,4 +25,5 @@ stages:
          X11Forwarding no
          AllowTcpForwarding no
          MaxAuthTries 3
+         Include /etc/ssh/sshd_config.d/*.conf
```

Comment:
Will the daemon run if `/etc/ssh/sshd_config.d/` doesn't exist or there are no files?

Reply:
Seems OK
```
harvester-node-1:~ # ls /etc/ssh/
moduli      ssh_host_dsa_key      ssh_host_ecdsa_key      ssh_host_ed25519_key      ssh_host_rsa_key      sshd_config
ssh_config  ssh_host_dsa_key.pub  ssh_host_ecdsa_key.pub  ssh_host_ed25519_key.pub  ssh_host_rsa_key.pub

harvester-node-1:~ # tail -n5 /etc/ssh/sshd_config
AllowAgentForwarding no
X11Forwarding no
AllowTcpForwarding no
MaxAuthTries 3
Include /etc/ssh/sshd_config.d/*.conf

harvester-node-1:~ # systemctl restart sshd.service
harvester-node-1:~ # systemctl status sshd.service
● sshd.service - OpenSSH Daemon
     Loaded: loaded (/usr/lib/systemd/system/sshd.service; enabled; vendor preset: enabled)
     Active: active (running) since Tue 2023-09-05 06:13:20 UTC; 7s ago
    Process: 29188 ExecStartPre=/usr/sbin/sshd-gen-keys-start (code=exited, status=0/SUCCESS)
    Process: 29190 ExecStartPre=/usr/sbin/sshd -t $SSHD_OPTS (code=exited, status=0/SUCCESS)
   Main PID: 29191 (sshd)
      Tasks: 4
     CGroup: /system.slice/sshd.service
             ├─ 14746 "sshd: rancher [priv]"
             ├─ 14771 "sshd: rancher@pts/0" ""
             ├─ 14772 -bash
             └─ 29191 "sshd: /usr/sbin/sshd -D [listener] 0 of 10-100 startups"
```